core/terraform/locals.tf

```diff
@@ -28,8 +31,9 @@ locals {
     "privatelink.cert.api.azureml.ms",
     "privatelink.notebooks.azure.net",
     "privatelink.postgres.database.azure.com",
-    "nexus-${var.tre_id}.${var.location}.cloudapp.azure.com",
     "privatelink.mysql.database.azure.com",
     "privatelink.azuredatabricks.net"
   ])
+
+  service_bus_namespace_fqdn = regex("(?:(?P<scheme>[^:/?#]+):)?(?://(?P<fqdn>[^/?#:]*))?(?::(?P<port>[0-9]+))?(?P<path>[^?#]*)(?:\\?(?P<query>[^#]*))?(?:#(?P<fragment>.*))?", azurerm_servicebus_namespace.sb.endpoint).fqdn
```

Comment:
![image](https://user-images.githubusercontent.com/1524451/228224024-9f8ab214-4354-4539-aa55-363c92e8f45e.png)

Reply:
is there any way we can pad this with comments so it's more obvious what it's expecting, the things it captures, and what it outputs?

Reply:
This is really the only conversation I have open that I think needs addressing.
If we can, I'd suggest some lines of comment above this regex to explain what it does and how it works, and what the expected inputs/outputs are.